.devcontainer/onCreateCommand.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Assumes the gh CLI is present in the default Codespaces image.
+gh extensions install github/gh-codeql
+gh codeql version # first command starts the download of the CodeQL CLI
+# Make codeql visible to VSCode by using https://github.com/github/gh-codeql#codeql-stub, since VS Code expects an executable called codeql instead of gh codeql
+gh codeql install-stub ~/.local/bin/
```

Comment:
Is `~/.local/bin` always on the `PATH`? or should we set the `codeQL.cli.executablePath` setting to point to `~/.local/bin/codeql`?

Reply:
We did consider this, but weren't sure if it was needed since it worked without this setting.

Reply:
By "find codeql without it" you mean the extension is able to run CodeQL-related commands, and did not install its own managed CLI? That suggests this folder is on the `PATH`.

Reply:
`~/.local/bin` seemed to be on the `PATH` indeed. The default of `/usr/local/bin` was owned by `root` and not `codespace` so it wasnt writable.
Let resolve the `PATH` vs `codeQL.cli.executablePath` in https://github.com/github/code-scanning/issues/9043 and I'll update this PR afterwards.

Reply:
We decided to go with `PATH` in https://github.com/github/code-scanning/issues/9043, so I think that resolves this conversation.